charts/kommander-operator/templates/post_install_kommandercore_hook.yaml

```diff
@@ -0,0 +1,106 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Chart.Name }}-installation
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": before-hook-creation
+    kubernetes.io/enforce-mountable-secrets: "true"
+secrets:
+  - name: {{ .Chart.Name }}-installation
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Chart.Name }}-installation
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": before-hook-creation
+    kubernetes.io/service-account.name: {{ .Chart.Name }}-installation
+type: kubernetes.io/service-account-token
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ .Chart.Name }}-installation
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": before-hook-creation
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Chart.Name }}-installation
+    namespace: {{ .Release.Namespace }}
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-installation
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+data:
+  install_crd.sh: |-
+    #!/bin/bash
+    set -eo pipefail
+
+     # wait until secret is created and has a token.
+    SECRET_NAME={{ .Chart.Name }}-installation
+    until kubectl get secrets -n {{ .Release.Namespace }} "$SECRET_NAME" -o jsonpath="{ .data.token }" &> /dev/null; do
+        echo "Waiting for the secret to be populated with credentials"
+        sleep 1
+    done
+
+    cat <<EOF | kubectl apply -f -
+    apiVersion: dkp.d2iq.io/v1alpha1
+    kind: KommanderCore
+    metadata:
+      name: kommander-core
+      namespace: kommander
```

Comment:
Should we remove the NS here? I thought this is a cluster resource. @palexster

Reply:
Yes, I fixed the KEP PR accordingly. Thanks!